docs/reference/remap/ipv6_to_ipv4.cue

```diff
@@ -11,6 +11,7 @@ remap: functions: ipv6_to_ipv4: {
 	]
 	internal_failure_reasons: [
 		"`ip` is not a valid IP address",
+		"`ipv6` address is not compatible with IPV4",
```

Comment:
```suggestion
		"`ip` address is not compatible with IPV4",
```

We should use the argument names here. Should we rename the argument name to `ipv6`?

Reply:
Technically it can be an ipv4 address, it just passes through the address. 

Or rather it should, I've just spotted that it is actually converting it to an ipv6 address and returning that, which it completely wrong. I'll raise an issue far that.